src/annotation/annotation.cpp

```
@@ -263,6 +263,46 @@ std::vector<DeleteLineAction> parse_delete_line_annotations(const std::string& t
   return actions;
 }
 
+// --- delegate ----------------------------------------------------------------
+
+/// Parse `<delegate>` XML annotations from LLM response text.
+std::vector<DelegateAction> parse_delegate_annotations(const std::string& text) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)